packages/fiori/src/ViewSettingsDialog.js

```diff
@@ -548,11 +549,14 @@ class ViewSettingsDialog extends UI5Element {
 		const _currentSortOrderSelected = this._currentSettings.sortOrder.filter(item => item.selected)[0],
 			_currentSortBySelected = this._currentSettings.sortBy.filter(item => item.selected)[0],
 			sortOrder = _currentSortOrderSelected && _currentSortOrderSelected.text,
-			sortBy = _currentSortBySelected && _currentSortBySelected.text;
-
+			sortDescending = !this._currentSettings.sortOrder[0].selected,
+			sortBy = _currentSortBySelected && _currentSortBySelected.text,
+			sortByItem = sortBy && this.initSortByItems.filter(item => item.text === sortBy)[0].associatedItem;
 		return {
 			sortOrder,
+			sortDescending,
 			sortBy,
+			sortByItem,
 			filters: this.selectedFilters,
```

Comment:
'confirm' and 'cancel' events 'detail' description don't inlcude newly added 'sortDescending' and 'sortByItem' properties, please fix this!